scripts/src/templates/dataset-mixed-content-modal.js

```diff
@@ -0,0 +1,21 @@
+export default (resourceUrl, originalDatasetUrl) => {
+    return `<div class="modal fade" tabindex="-1" role="dialog" id="mixed-content-warning-modal">
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">Mixed content warning</h4>
+      </div> 
+      <div class="modal-body">
+        <p>This resource link is only served over HTTP. As such, we can not link to it as most modern browsers do not allow direct links from HTTPS secured sites to HTTP sites for security reasons.</p>
+        <p>To access the resource, please copy the following URL and paste it into the address bar of your browser:</p>
+        <pre>${resourceUrl}</pre>
```

Comment:
wonder if we can add a "copy all" / "copy to clipboard" button here? It's too easy to make a mistake here and not copy the whole url.

Reply:
![mixed-content-clipboard](https://user-images.githubusercontent.com/46202639/221021284-23130daf-0f73-4299-b0be-2ba10010f342.gif)
Good shout. Have implemented a Copy button.

Reply:
loving the screen recording - how handy!